docs/contributing/guidelines-for-pull-request.md

```diff
@@ -2,30 +2,38 @@
 ##  Before pull request
 * Fork our [monorepo](https://github.com/shopsys/shopsys)
 * Create your branch
-      
-        git branch branch-name
-      
-* Make change and follow [commit message guidelines](./guidelines-for-creating-commits.md)
+
+```
+git branch branch-name
+```
+
+* Create contribution and follow [commit message guidelines](./guidelines-for-creating-commits.md)
```

Comment:
Create contribution sound weird to me. Make contribution sounds a little bit better. Maybe something like "Make desired changes in code and then follow .... ?

Reply:
And update even changelog please